web/src/app/administration/components/access-items-management/access-items-management.component.html

```diff
@@ -1,177 +1,136 @@
-                </button>
-              </th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-              <th></th>
-            </tr>
-          </thead>
-
-          <!-- ACCESS ITEMS GROUPS -->
-          <tbody formArrayName="accessItemsGroups">
-            <tr class="table__access-item-groups"
-              *ngFor="let accessItem of accessItemsGroups.controls; let index = index;"
-              [formGroupName]="index.toString()">
-              <td colspan="2">
-                <label class="wrap">{{accessItem.value.workbasketKey}}</label>
-              </td>
-              <td *ngIf="(accessItemsCustomization$ | async)?.accessId.lookupField else accessIdInput" colspan="2"
-                class="text-align text-width taskana-type-ahead" style="padding-top: 0;">
-                <div>
-                  <taskana-shared-type-ahead formControlName="accessId" placeHolderMessage="* Access id is required"
-                    [validationValue]="toggleValidationAccessIdMap.get(index)"
-                    [displayError]="!isFieldValid('accessItem.value.accessId', index)" [disable]=true>
-                  </taskana-shared-type-ahead>
-                </div>
-              </td>
-              <ng-template #accessIdInput>
-                <td colspan="2" class="text-align text-width">
-                  <div>
-                    <label>
-                      <input type="text" class="form-control" formControlName="accessId" placeholder="{{accessItem.invalid?
+              </tr>
+            </thead>
+            <tbody formArrayName="accessItemsGroups">
+              <tr class="table-access-item__content"
+                *ngFor="let accessItem of accessItemsGroups.controls; let index = index;"
+                [formGroupName]="index.toString()">
+                <td></td>
+                <td colspan="1" class="text-align text-width taskana-type-ahead">
+                  <label class="wrap">{{accessItem.value.workbasketKey}}</label>
+                </td>
+                <td *ngIf="(accessItemsCustomization$ | async)?.accessId.lookupField else accessIdInput"
+                  class="text-align text-width taskana-type-ahead">
+                  <label matTooltip="{{accessItem.value.accessId}}" class="wrap">{{accessItem.value.accessName}}</label>
+                </td>
+                <ng-template #accessIdInput>
+                  <td colspan="1" class="text-align text-width">
+                    <div>
+                      <label>
+                        <input type="text" class="form-control" formControlName="accessId" placeholder="{{accessItem.invalid?
```

Comment:
'form-control' is bootstrap. There are input fields from material: https://material.angular.io/components/input/overview